errai-demos/errai-security-demo/pom.xml

```diff
@@ -310,6 +311,14 @@
           <extraJvmArgs>-Xmx1G -XX:CompileThreshold=7000 -Derrai.jboss.home=${errai.jboss.home} -Derrai.dev.context=${errai.dev.context}</extraJvmArgs>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-deploy-plugin</artifactId>
+        <version>${maven.deploy.plugin.version}</version>
+        <configuration>
+          <skip>true</skip>
+        </configuration>
```

Comment:
Sorry for the late reply, I didn't spot this earlier for some reason.. @psiroky AFAIK skipping deployment does fail the build in Brew. But I also noticed we were doing this in the other demos, maybe this will be OK. 

I'm not sure if there's some specifics to when Brew will and will not throw the unknown file in build root error.